interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -847,19 +835,18 @@ function postcalendar_admin_testSystem()
     }
     array_push($infos, array('smarty compile dir',  $info));
 
-    $header = <<<EOF
-	<head></head>
-	<body>
-EOF;
+    $header = "<head><title>" . xlt("Diagnostics") . "</title></head><body>";
     $output .= $header;
-    $output  = postcalendar_adminmenu("testSystem");
-    $output .= '<div class="container table-responsive"><table class="table table-bordered table-striped  "><thead>';
+    $output .= '<div class="container mt-3"><div class="row"><div class="col-sm-12"><div class="page-header clearfix">';
+    $output .= '<h2>'. xlt('Diagnostics') . '</h2>';
```

Comment:
lets do an acl_check here to only allow users with admin->super privileges to see this. If they don't then would display message "Not Authorized" instead. To do this requires the following use at top of script:
```
use OpenEMR\Common\Acl\AclMain;
```
Then can check if have access by running the following function:
```
if (AclMain::aclCheckCore('admin', 'super')) {
```